style.css

```
@@ -0,0 +1,123 @@
+* {
+  padding: 0;
+  margin: 0;
+  font-family: "Arial Black ", Arial, Helvetica, sans-serif;
+  box-sizing: border-box;
+}
+
+nav {
+  background-color: #498FE2;
+  min-height: 10vh;
+  min-width: 100vw;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+nav > h1 {
+  color: white;
+  font-weight: bolder;
+  font-size: 1.75rem;
+}
+
+main {
+  background-color: #f0f3f7;
+  min-width: 100vw;
+  min-height: 82vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.content-wrapper {
+  background-color: #f0f3f7;
+  display: flex;
+  flex-direction: row;
+  align-items: stretch;
+  justify-content: center;
+  min-width: 80vw;
+  margin-top: 10vh;
+  margin-bottom: 10vh;
+}
+
+.note {
+  background-color: #fffffe;
+  padding: 1.25rem;
+  flex: 2;
+  box-shadow: 0rem 0.25rem 0.5rem rgba(0, 0, 0, 0.1);
+  margin-right: 2rem;
+}
+
+.resources {
+  background-color: #f7f8fb;
+  padding: 1.25rem;
+  flex: 1;
+  box-shadow: 0rem 0.25rem 0.5rem rgba(0, 0, 0, 0.1);
+}
+
+.resources > h2,
+.note > h2 {
+  font-weight: bolder;
+  margin-bottom: 2vh;
+  margin-top: 2.5vh;
+  color: #3a3a3a;
+  font-size: 1.75rem;
+}
+
+.elements {
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+}
+
+.elements > a {
+  text-decoration: none;
+  margin-bottom: 1.5vh;
+  color: #6b3a99;
+  font-weight: bold;
+}
+
+form {
+  gap: 0.9375rem;
+  display: flex;
+  flex-direction: column;
+}
+
+#title, #content {
+  border: 1px solid #b9b8b8;
+  border-radius: 0.3125rem;
+  padding: 0.3125rem;
+  margin-bottom: 0.3125rem;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+label, input, textarea {
+  margin: 0;
+  padding: 0;
+}
+
+.form-button {
```

Implement BEM for all classes
```suggestion
.form__button {
```